rca/static_src/sass/components/_tab-item.scss

```diff
@@ -27,6 +27,16 @@
         @include underline-hover-target-reset();
     }
 
+    &--highlight {
+        // Highlighted butons will stay orange both in dark and light bg.
+        color: $color--orange !important;
```

Comment:
How come `!important` is needed?

Reply:
It's so it'll override the colors specified below.

We could do something like this:

```
.bg--dark & {
        color: $color--meta-darkbg;

        &:hover,
        &.active {
            color: $color--white;
        }

        &.link--tertiary {
            color: $color--meta-darkbg;

            &:hover {
                color: $color--white;
            }
        }

        &#{$root}--highlight {
            color: $color--orange;

            &:hover,
            &.active {
                color: $color--orange;
            }
        }
    }
```

But then we'd have to repeat it for `bg-light` as well.

Reply:
Ah ok, that is a fair amount of code to have to reproduce.